src/main/java/frc/robot/subsystems/shooter/ShooterIOSim.java

```diff
@@ -93,4 +90,9 @@ public void setFlywheelVoltage(final double left, final double right) {
     leftFlywheelSim.setInput(MathUtil.clamp(left, -12, 12));
     rightFlywheelSim.setInput(MathUtil.clamp(right, -12, 12));
   }
+
+  @Override
+  public void resetPivotPostion(Rotation2d rotation) {
```

Comment:
Was this missing a bug?

Reply:
having it be a noop shouldve been fine? the point of the method is to reset the encoder to match physical hardware whereas this always has exact knowledge of its state so it shouldnt matter.